src/mysql_protocol.erl

```diff
@@ -1018,11 +1018,18 @@ parse_packet_header(<<PacketLength:24/little-integer, SeqNum:8/integer>>) ->
 add_packet_headers(PacketBody, SeqNum) ->
     Bin = iolist_to_binary(PacketBody),
     Size = size(Bin),
-    SeqNum1 = (SeqNum + 1) band 16#ff,
-    %% Todo: implement the case when Size >= 16#ffffff.
-    if Size < 16#ffffff ->
-        {[<<Size:24/little, SeqNum:8>>, Bin], SeqNum1}
-    end.
+    add_packet_headers(Bin, Size, <<>>, SeqNum).
+
+%% @doc Manages all sizes of datas to send even with sizes greater than 0xffffff (16#ffffff).
```

Comment:
This comment looks like a commit message, something for the change log. For describing the function, something in line with "helper for add_packet_headers/2" would be more useful, I think.